themes.json

```diff
@@ -350,5 +350,37 @@
         "/posts/instructions.md"
       ]
     }
+  },
+  {
+    "id": "pro-file",
+    "name": "pro-file",
+    "description": "A theme to create a simple resume website.",
+    "tags": ["cv", "resume"],
+    "lume_version": 2,
+    "author": {
+      "name": "Nap Joseph Calub",
+      "url": "https://napjose.ph"
+    },
+    "repo": "https://github.com/wewillcraft/pro-file",
+    "demo": "https://pro-file.willcraft.software",
+    "screens": [
+      {
+        "desktop": "/screens/pro-file/desktop.png",
+        "mobile": "/screens/pro-file/mobile.png"
+      }
+    ],
+    "module": {
+      "name": "pro-file",
```

Comment:
The `name` is used to create the variable to import the theme. In your case it would be:

```js
import pro-file from "https://deno.land/x/pro_file/main.ts";
```

This will fail because variables only can have alphanumeric characters. It must be changed to something valid for a variable name like `profile` or `pro_file`.

Reply:
Noted on this!

Reply:
Changed it to `pro_file` instead.